src/ApiClients/ProductApi/ProductApiClient.php

```diff
@@ -1,15 +1,22 @@
 <?php
 
 
-namespace SchierProducts\SchierProductApi;
+namespace SchierProducts\SchierProductApi\ApiClients\ProductApi;
+
+use SchierProducts\SchierProductApi\ApiClients\ProductApi\Resources\Product;
+use SchierProducts\SchierProductApi\Client;
+use SchierProducts\SchierProductApi\Collection;
+use SchierProducts\SchierProductApi\Exception;
+use SchierProducts\SchierProductApi\Resources;
+use SchierProducts\SchierProductApi\Service;
 
 /**
  * Client used to send requests
  *
  * @ProductApiClient
  * @package SchierProducts\SchierProductApi
- * @property \SchierProducts\SchierProductApi\Service\ProductTypeService $productTypes
- * @property \SchierProducts\SchierProductApi\Service\ProductService $products
+ * @property \SchierProducts\SchierProductApi\ApiClients\ProductApi\Service\ProductTypeService $productTypes
+ * @property \SchierProducts\SchierProductApi\ApiClients\ProductApi\Service\ProductService $products
  * @property \SchierProducts\SchierProductApi\Service\ItemCollectionService $collections
```

Comment:
This can be removed as it is no longer a feature.

```suggestion
 * @property \SchierProducts\SchierProductApi\Service\ItemCollectionService $collections
```